modules/efk-logging-curator-schedule.adoc

```diff
@@ -5,16 +5,26 @@
 [id='efk-logging-curator-schedule_{context}']
 = Configuring the Curator schedule
 
-You can specify the schedule for Curator in cron format.
+You can specify the schedule for Curator using the the `logging-curator` configuration map
+created by the Cluster Logging Operator installation.
 
 [NOTE]
 ====
 The time zone is set based on the {product-title} master node.
```

Comment:
Sorry, this is my bad. I checked again and it actually uses the time zone configure on the host, where curator pod is scheduled to. It is unlikely that it would be master node. Just a node/host in general.

Reply:
@josefkarasek 
Changed to _The time zone is set based on the {product-title} node host._